src/index.css

```diff
@@ -0,0 +1,93 @@
+.moving-navbar {
+  box-shadow: 1px 2px 3px rgba(0,0,0,0.45);
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 200;
+  height: 50px;
+  transition: top .2s ease-in-out;
+  align-items: center;
+  background-color: #49505C;
+  box-sizing: content-box;
+  color: #d8d8da;
+  display: flex;
+  flex-flow: row nowrap;
+  font-family: 'Sofia Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  height: 66px;
+  line-height: 65px;
+  margin: 0;
+  padding: 0 10px;
+  pointer-events: none;
+  position: relative;
+  text-transform: uppercase;
+  top: 0;
+  width: auto;
+  display: flex;
+  justify-content: space-between;
+}
+.moving-navbar-button {
+  padding: 0 15px;
+  color: #d8d8da;
+  font-size: 14px;
+  display: inline-block;
+  height: 50px;
+  line-height: 50px;
+  box-sizing: border-box;
+  padding: 0 20px;
+}
+.countdown-container {
+  display: flex;
+  flex-shrink: 0;
+  font-size: 12px;
+  margin-right: 0.5em;
+}
+.blocky-countdown {
+  font-size: .9em;
+  padding: 0.1em 0.25em 0.5em;
+  background-color: rgba(10,10,10,0.5);
+  color: white;
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 0.2em;
+  width: 3em;
+  line-height: 1.3;
+  margin: 0 0.2em;
+}
+.countdown-number {
+  font-weight: bold;
+  font-size: 1.75em;
+  display: block;
+  line-height: 1.3;
+}
+.countdown-word {
+  font-size: 1em;
+  line-height: 1;
+  opacity: 0.75;
+  text-transform: lowercase;
+}
+.no-style {
+  text-transform: none;
+}
```

Comment:
What's the purpose of this rule?

Reply:
This removes the transforms being used in the parent class which makes everything upper-case.